src/index.html

```diff
@@ -0,0 +1,37 @@
+<!DOCTYPE html>
```

Comment:
**Kindly work on theses changes:**
- Following your [general requirements](https://github.com/microverseinc/curriculum-transversal-skills/blob/main/documentation/articles/professional_repo_rules.md) your project it's missing the information the README file information. If you want to have a template click [here](https://github.com/microverseinc/readme-template).